src/components/Routing/ListItemLink.tsx

```diff
@@ -30,7 +30,7 @@ interface ExternalProps extends BaseProps {
 export const ListItemLink = forwardRef<HTMLAnchorElement, ListItemLinkProps>(
   ({ external, to, children, ...props }, ref) => {
     const path = typeof to === 'string' ? to : { path: to.pathname! };
-    const active = useMatch(path);
+    const active = useMatch(`${path}/*`);
```

Comment:
I believe match has an `end` option. It would be better to use that than assume things about the path string.

Reply:
This is a weird case where we actually want to match the beginning of the string instead of the end in the sidebar. Instead of passing in multiple booleans for if we want to match the beginning or end we can either add in some prop called pathToMatch that can be used to tell if the paths match or just pass in whether the component is active in from the outside.

Reply:
Does 
```tsx
    const path = typeof to === 'string' ? to : to.pathname!;
    const active = useMatch({ path, end: false });
```
do the same thing as what you have above? Does it make the sidebar selection work correctly?

Reply:
To be clearer, `end` is true by default. We could probably default to false here and add a `exact` prop that would set end to true.